### ISSUE TITLE
Prepayment Schedules Customization - MDTL UK BC Tenant-CAS-01409-G5C6W3

### ISSUE THREAD

**Issue by Prakash3K** [3ktech/General#654]
**Approved time 3 Hours+4 Hours = 7 Hours**

[Detail Trial Balance (8).xlsx](https://github.com/3ktech/General/files/12461109/Detail.Trial.Balance.8.xlsx)

**Detail Trial Balance (4, Report Request)**

**Comment by TabrejAlam3K** [comment]
Mr. @3ktech - Can you please share the time effort for this

We have already done the same things for India BC Tenant

CC : @audhi3k @Shasthara @narayana3kt

**Comment by TabrejAlam3K** [comment]
Hi Mr. @3ktech - Please assign the technical who will work upon.

CC : @audhi3k

**Comment by audhi3k** [comment]
Mr Karthik @3ktech share the time estimate and align the case to technical

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - Please update the time effort first before starting the work.

CC : @audhi3k  @3ktech

**Comment by TabrejAlam3K** [comment]
User will provide the new credential to logon into the BC.

I was connect with Mr. Boopathy, He will provide the Password soon.

CC : @audhi3k @sruthi1017

**Comment by TabrejAlam3K** [comment]
We are still waiting for Credential and Approval for the same.

CC : @audhi3k

**Comment by TabrejAlam3K** [comment]
Approval received from the client side, Still waiting for login credentials.

CC : @audhi3k @sruthi1017

**Comment by 3ktech** [comment]
waiting for the report id information @TabrejAlam3K , @niranjand22 , @audhi3k

**Comment by audhi3k** [comment]
the report was developed by our technical, we can get from also Mr Karthik @3ktech

**Comment by 3ktech** [comment]
Detailed Trial Balance Report - Merit #82
@sruthi1017 , @sandhya3k  has done this task in IN tenant. please have a word with her and do the same in UK tenant

**Comment by audhi3k** [comment]
@3ktech , why this is in Internal QA - wherein there is no comments

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - Make sure this changes should be reflecting in the Sandbox under the Merit Data & Technology Limited Company.

CC : @audhi3k @3ktech

**Comment by TabrejAlam3K** [comment]
![Image](https://github.com/3ktech/General/assets/116171982/c816f9cd-f3e1-48f5-9fd5-1f1453e99218)

**Comment by TabrejAlam3K** [comment]
![Image](https://github.com/3ktech/General/assets/116171982/e76d2391-66bc-4a2b-8623-7ed0630126f2)

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - Please update you have deployed the changes into the Production.

**Comment by TabrejAlam3K** [comment]
As per our discussion over team chat Ms. Swathi is busy in another task, She will connect tomorrow post lunch.

**Comment by Jayanth3KT** [comment]
Hi @TabrejAlam3K Can I know the current status of this ticket.

CC: @Shasthara @audhi3k

**Comment by TabrejAlam3K** [comment]
Hi Mr. @Jayanth3KT - Waiting for Customer Response on.

CC : @audhi3k @Shasthara

**Comment by TabrejAlam3K** [comment]
Waiting for Customer Confirmation

CC : @audhi3k @Jayanth3KT @Shasthara

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - Please move the below customization from India tenant to UK Tenant.

1. G/L Deferral Summary Cust - 50110
2. Purchasing Deferral Summary Cust - 50111

CC : @audhi3k @3ktech

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - As per our discussion please publish the above report in UK Tenant, First in Sandbox then Production.

CC : @audhi3k @3ktech

**Comment by TabrejAlam3K** [comment]
Following report has been deployed in the Production Environment. I have also send an email to the Client for the same for UAT.

1. G/L Deferral Summary Cust - 50110
2. Purchasing Deferral Summary Cust - 50111

CC : @audhi3k @Shasthara @Jayanth3KT

**Comment by TabrejAlam3K** [comment]
Hi Mr. @Jayanth3KT - Close the ticket from the support portal. I am closing the git ticket. Client asked for closure.

CC : @audhi3k @Shasthara
